test/SPIRV/OpLine.ll

```diff
@@ -0,0 +1,169 @@
+;__kernel void foo(){}
+;int bar(int a) {
+;    a += 3;
+;    int b = 10;
+;    while (b) {
+;        if (a > 4)
+;            b = a - 1;
+;        else  {
+;            b = a + 1;
+;            a = a *3;
+;        }
+;    }
+;    return b;
+;}
+; RUN: llvm-as < %s | llvm-spirv -spirv-text | FileCheck %s
```

Comment:
I have made the test bi-way. It checks line information correctness after bi-way translation.